### PR TITLE
code: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** latest

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]
Look good to me.

File: pages/common/code.md [hunk 1]
```diff
@@ -3,30 +3,34 @@
 > Visual Studio Code.
 > More information: <https://github.com/microsoft/vscode>.
 
-- Open VS Code:
+- Start the editor:
```
Comment (CleanMachine1) [inline pages/common/code.md:6]:
```suggestion
- Start VS Code:
```

File: pages/common/code.md [hunk 2]
```diff
@@ -3,30 +3,34 @@
...
 
 `code`
 
-- Open the current directory in VS Code:
```
Comment (CleanMachine1) [inline pages/common/code.md:10]: This example is more useful and should be kept.

File: pages/common/code.md [hunk 3]
```diff
@@ -3,30 +3,34 @@
...
+`code --new-window {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Open a file or directory in a new VS Code window:
+- Install/uninstall a specific extension:
 
-`code --new-window {{path/to/file_or_directory}}`
+`code --{{install|uninstall}}-extension {{publisher.extension}}`
 
-- Compare two files in VS Code:
+- Print installed extensions:
```
Comment (CleanMachine1) [inline pages/common/code.md:26]: Lets remove this and add the `code .` example.

File: pages/common/code.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/common/code.md:10]: I want make pages more general and demonstrate as much as possible command syntax. 😄 That's why I used:

```md
- Open specific files/directories:

`code {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```

`code` accepts not just one path but several ones.

Comment (CleanMachine1) [reply pages/common/code.md:10]: Personally, `code .` is the only command for code I run.

I'd imagine that not everyone knows that `.` is a valid, whilst im sure that most people will infer that you can enter another directory if you can do you current directory.

File: pages/common/code.md [hunk 4]
```diff
@@ -1,32 +1,36 @@
 # code
 
-> Visual Studio Code.
+> Visual Studio Code is a lightweight but powerful source code editor which runs on your desktop and is available for Windows, macOS and Linux.
```
Comment (CleanMachine1) [inline pages/common/code.md:3]:
```suggestion
> Visual Studio Code is a lightweight and powerful code editor .
```

Lots of extra fluff not needed.

**Review by einverne — APPROVED** [review]
Looks good.

**Comment by CleanMachine1** [comment]
Please don't merge unless all comments are addressed. Also let other maintainers merge as stated in the guides

**Comment by EmilySeville7cfg** [comment]
Okay, sorry...

**Pull request merged**